rest/stargate.go

```diff
@@ -132,7 +127,13 @@ func handleStargateTxsSearch(c *gin.Context, pool *pgxpool.Pool) {
 		}
 	}
 
-	txResponses, err := db.QueryTxs(conn, events, height, limit, offsetInTimesOfLimit, order)
+	var txResponses []*types.TxResponse
+	if keyword := q.Get("q"); keyword != "" {
+		txResponses, err = db.QueryTxsWithKeyword(conn, events, limit, offsetInTimesOfLimit, order, keyword)
+	} else {
+		txResponses, err = db.QueryTxs(conn, events, height, limit, offsetInTimesOfLimit, order)
+	}
```

Comment:
Just an opening discussion, is it possible to use a single function `QueryTxs()` to serve all kinds of query?

Reply:
Yes, I updated that to amino and stargate. However, it doesn't work with QueryISCN. It seems like there are different system of index. @williamchong  Please check my last commit.

Reply:
Removed all full-text search function